sovrin_node/server/config_req_handler.py

```diff
@@ -63,6 +63,12 @@ def validate(self, req: Request, config=None):
             # present
             status = self.upgrader.statusInLedger(req.operation.get(NAME),
                                                   req.operation.get(VERSION))
+            if status == START and action == START:
+                raise UnauthorizedClientRequest(
```

Comment:
Sure, we can make it a more appropriate message.  The ticket suggested UnauthorizedClientRequest, I will change and re-submit this morning.